Doc/library/selectors.rst

```diff
@@ -63,6 +63,11 @@ constants below:
    +-----------------------+-----------------------------------------------+
    | :const:`EVENT_WRITE`  | Available for write                           |
    +-----------------------+-----------------------------------------------+
+   | :const:`EVENT_URGENT` | Available for urgent data to read             |
+   +-----------------------+-----------------------------------------------+
```

Comment:
Hum, since KqueueSelector doesn't support it. It would be nice to list selectors which support it or not. Maybe just mention that KqueueSelector doesn't support urgent events?